evm/gas-service.js

```diff
@@ -0,0 +1,414 @@
+'use strict';
+
+const { ethers } = require('hardhat');
+const {
+    getDefaultProvider,
+    utils: { formatEther, parseEther },
+    Contract,
+    BigNumber,
+} = ethers;
+const { Command, Option } = require('commander');
+const {
+    printInfo,
+    printWalletInfo,
+    printWarn,
+    mainProcessor,
+    prompt,
+    getGasOptions,
+    wasEventEmitted,
+    isValidAddress,
+    validateParameters,
+} = require('./utils');
+const { addBaseOptions } = require('./cli-utils');
+const IAxelarGasService = require('@axelar-network/axelar-gmp-sdk-solidity/interfaces/IAxelarGasService.json');
+const { getWallet } = require('./sign-utils');
+
+async function getGasInfo(env, chain, destinationChains) {
+    const apiUrl = env === 'testnet' ? 'https://testnet.api.axelarscan.io/gmp/getFees' : 'https://api.axelarscan.io/gmp/getFees';
+
+    return Promise.all(
+        destinationChains.map((destinationChain) => {
+            let feeType = 0;
```

Comment:
```suggestion
            let gasEstimationType = 0;
```
to match the contract name